contracts/veLON.sol

```diff
@@ -42,6 +43,114 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
         poolPointHistory[0].ts = block.timestamp;
     }
 
+    function vBalanceOf(uint256 _tokenId) external view override returns (uint256) {
+        return _vBalanceOfAtTime(_tokenId, block.timestamp);
+    }
+
+    function vBalanceOfAtTime(uint256 _tokenId, uint256 _t) external view override returns (uint256) {
+        require(_t <= block.timestamp, "Invalid timestamp");
```

Comment:
Does it need to forbid user querying the past voting power? Perhaps users just want to calculate the voting power by themselves?

Reply:
Actually it avoid `_t` is in the future. Historical vBalance querying is available.

Reply:
Oh sorry, I misunderstand the symbol haha